substrate/bin/utils/chain-spec-builder/README.md

```diff
@@ -0,0 +1,117 @@
+# Chain Spec Builder
+
+Substrate's chain spec builder utility.
+
+A chain-spec is short for `chain-specification`. See the [sc-chain-spec](https://crates.io/crates/sc-chain-spec)
+for more information.
+
+_Note:_ this binary is a more flexible alternative to the `build-spec` subcommand, contained in typical Substrate-based nodes.
+This particular binary is capable of interacting with [sp-genesis-builder](https://docs.rs/sp-genesis-builder/latest/sp_genesis_builder/index.html)
+implementation of any provided runtime allowing to build chain-spec JSON files.
+
+See [ChainSpecBuilderCmd](https://docs.rs/staging-chain-spec-builder/6.0.0/staging_chain_spec_builder/enum.ChainSpecBuilderCmd.html)
+for a list of available commands.
+
+## Installation
+
+```bash
+cargo install staging-chain-spec-builder
+```
+
+_Note:_ `chain-spec-builder` binary is published on [crates.io](https://crates.io) under
+[staging-chain-spec-builder](https://crates.io/crates/staging-chain-spec-builder) due to a name conflict.
+
+## Usage
+
+### Generate chains-spec using default config from runtime
+
+Query the default genesis config from the provided `runtime.wasm` and use it in the chain
+spec.
+
+```bash
+chain-spec-builder create -r runtime.wasm default
+```
+
+_Note:_ [GenesisBuilder::get_preset](https://docs.rs/sp-genesis-builder/latest/sp_genesis_builder/trait.GenesisBuilder.html#method.get_preset)
```

Comment:
IMO this would look better:
```suggestion
_Note:_ [`GenesisBuilder::get_preset`](https://docs.rs/sp-genesis-builder/latest/sp_genesis_builder/trait.GenesisBuilder.html#method.get_preset)
```

Reply:
[37e43c0](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/37e43c03090ee56e14c5df4da2c42945e6bcc08e)